package.json

```diff
@@ -30,7 +29,7 @@
     "grunt-contrib-watch": "0.6.1",
     "grunt-contrib-clean": "0.5.0",
     "grunt-mocha": "0.4.7",
-    "contextify": "0.1.9",
+    "contextify": "^0.1.14",
```

Comment:
nice - i guess this was the install bug you were seeing?

must have been annoying to hunt down